config.json

```diff
@@ -77,7 +77,11 @@
             "active": true
         },
         "worksheets": {
-            "active": true
+            "active": true,
+            "features": {
+                "nlq":true,
+                "max_count_per_day": 25
```

Comment:
nit: move max_count_per_day inside the nlq feature? It makes it longer, but it is more organised
```
"nlq": {
    "active": true,
    "max_count_per_day": 25
},
```

Reply:
you are right I will change